tests/integration/devfile/cmd_devfile_build_images_test.go

```diff
@@ -79,6 +79,9 @@ var _ = Describe("odo devfile build-images command tests", func() {
 
 		BeforeEach(func() {
 			helper.CopyExample(filepath.Join("source", "nodejs"), commonVar.Context)
+			helper.CopyExample(filepath.Join("source", "devfiles", "nodejs", "project", "kubernetes",
```

Comment:
Should odo copy these dependencies when running `odo init --devfile-path`? @kadel WDYT?

Reply:
+1. Indeed, that would be a nice feature, I think.

Reply:
For reference, #5713 has been created for this point.